assets/src/dashboard/app/serializers/savedTemplates.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { migrate, DATA_VERSION } from '../../../edit-story/migration/migrate';
+import { toUTCDate } from '../../../date';
+
+// TODO: this is mostly placeholder to generally lend shape to saved templates that matches stories. We do no have stories formatted and returned in saved template API yet
+export default function reshapeSavedTemplateObject(originalSavedTemplateData) {
+  const {
+    id,
+    title,
+    modified_gmt,
+    status,
+    date_gmt,
+    link,
+    story_data: storyData = {},
+  } = originalSavedTemplateData;
+
+  if (!Array.isArray(storyData?.pages) || !id || storyData.pages.length === 0) {
+    storyData.pages = [];
```

Comment:
These will need a single page object with an `animations` and `elements` array. Those can be empty. Otherwise the story provider will throw errors.

Reply:
My thought here was just in getting the saved templates view to render, I had intentionally left out any deeper placeholder data structure to serve as a reminder that this isn't done - but you're right that having a console sans warning logs is really nice. I updated to have animations and elements as well as a placeholder id now that pages exists so that we don't get additional warnings about ids missing within the grid. Thanks for the suggestion.